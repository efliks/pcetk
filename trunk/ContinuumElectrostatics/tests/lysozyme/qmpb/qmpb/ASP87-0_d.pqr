../0/site_ASP_87_d.pqr