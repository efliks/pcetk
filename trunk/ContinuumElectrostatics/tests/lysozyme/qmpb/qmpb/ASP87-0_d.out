Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named ASP87-0_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (-3.757, 7.778, 16.234) 101 1
Finer level:    (-3.757, 7.778, 16.234) 101 0.5
Finest level:   (-3.757, 7.778, 16.234) 101 0.25


Self energy of ASP87-0_d in three dielectric environment = 409.825
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of ASP87-0_d with background charge set in three dielectric environment = -4.49606
(probably in kcal/mole)



Interaction energies of ASP87-0_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.39401
0   1 : 0.00709862
1   0 : -0.00869788
1   1 : 0.24747
2   0 : -0.0601649
2   1 : -0.000314154
3   0 : -1.03135
3   1 : 0.218064
3   2 : -0.113746
3   3 : 1.22006
4   0 : -0.00944495
4   1 : 0.0330672
5   0 : 0.000881786
5   1 : 0.059082
6   0 : 0.00168669
6   1 : 0.0140047
7   0 : -0.0192274
7   1 : -0.000251944
8   0 : -0.00746907
8   1 : 0.0312764
9   0 : 0.000452059
9   1 : 0.00643098
10   0 : -0.00435312
10   1 : 0.0230967
11   0 : 0.000841563
11   1 : 0.0477139
12   0 : -0.00335343
12   1 : 0.0277033
13   0 : 501.867
13   1 : 819.649
14   0 : -0.1686
14   1 : 0.000830517
15   0 : -0.0539974
15   1 : -0.000169511
16   0 : 0.00171118
16   1 : 0.022961
17   0 : -0.00380319
17   1 : -1.6985e-05
18   0 : 0.000113413
18   1 : 0.00507426
Total runtime: 7.37 s user 0.06 s system 
